openssh/files/ssh_config

```diff
@@ -1,29 +1,71 @@
 {%- from "openssh/map.jinja" import client with context %}
 {%- from "linux/map.jinja" import network with context %}
```

Comment:
Is this still needed? I would prefer removing cross-formula dependencies that are not needed anymore.

Reply:
network is still used:
- https://github.com/salt-formulas/salt-formula-openssh/pull/10/files#diff-41b97c7794214fea83bdeef3cf284554R9
- https://github.com/salt-formulas/salt-formula-openssh/pull/10/files#diff-4f7b47e0118a01d2898261d8a8538c72L19